rust/src/measurements/rappor/make_rappor.tex

```diff
@@ -0,0 +1,167 @@
+\documentclass{article}
+\input{../../lib.sty}
+
+
+\title{\texttt{fn make\_rappor}}
+\author{Abigail Gentle}
+\begin{document}
+
+\maketitle
+
+
+
+\contrib
+
+Proves soundness of \rustdoc{measurements/fn}{make\_rappor} in \asOfCommit{mod.rs}{304ef9c2}.
```

Comment:
This is really a process question for @Shoeboxam : After the squash merge, is it safe to continue to reference a commit from a feature branch? I think github will keep it around, but if we auto-delete the branch now, I don't know that future checkouts would have this history. 

If we do need this, it might be a reason to turn auto-delete off again.

Reply:
Ultimately these should point to commits in main. I was thinking updating these hashes would be part of a stabilization PR down the line that also drops contrib.